ports/rp2/modrp2.c

```diff
@@ -65,7 +65,12 @@ static bool __no_inline_not_in_flash_func(bootsel_button)(void) {
 
     // The HI GPIO registers in SIO can observe and control the 6 QSPI pins.
     // The button pulls the QSPI_SS pin *low* when pressed.
-    bool button_state = !(sio_hw->gpio_hi_in & (1 << CS_PIN_INDEX));
+    #if PICO_RP2040
+        #define CS_BIT (1u << 1)
```

Comment:
Can this be `1u << CS_PIN_INDEX`?

Reply:
Any idea why `CS_PIN_INDEX` even exists at all? It's inherited from the original example code, so readability I would guess.

I'll move it to a define so that moving the `#if` block is possible.